pkg/build/types/image_configuration.go

```diff
@@ -79,3 +80,19 @@ func (ic *ImageConfiguration) ValidateServiceBundle() error {
 
 	return nil
 }
+
+func (ic *ImageConfiguration) Summarize() {
```

Comment:
`entrypoint` and `accounts` could be hidden if empty :thinking: . Thoughts ?

Reply:
Yes, do it.

Reply:
Fixed